v2/memory.go

```diff
@@ -19,7 +19,7 @@ package v2
 type Memory struct {
 	Swap *int64
 	Max  *int64
-	High *int64
```

Comment:
no need to remove High?

Reply:
Is this a question? coz I removed it;)

Reply:
Or you wanted to say that we still need `High`? If so, then what for? what we will store there?

Reply:
Probably we don't have usecase atm, but the package should be able to handle all features supported by the kernel.

Reply:
Ok, I will put it back then.

Reply:
Done